### ISSUE TITLE
[Suggestion] Command/gui option for admins to remove item/pokemon

### ISSUE THREAD

**Issue by OfficialSqu1dz** [NickImpact/GTS#71]
Is there a possibility we could get a option with gts where admins can run a command/click on a button to take someones pokemon or item off of gts and give it back to them/put it back in their inventory? As constantly using adding money with an economy plugin and then removing the pokemon/item and paying them back gets slightly annoying after a while.

**Comment by NickImpact** [comment]
Can’t tell what you mean because it looks like the logic of this ticker went 3 different directions, but admin functionality is already planned to come back once my schedule opens up for it.

**Comment by OfficialSqu1dz** [comment]
I mean like let's say there's an option within the GUI that shows the item/poke you're about to buy that allows you to remove the listing from the GTS

**Comment by NickImpact** [comment]
Yes players can already do this by viewing their own listings.

**Comment by OfficialSqu1dz** [comment]
I meant for admins to do this to regular players, for example a player undersells a legendary poke, and the admin can remove the listing from the GTS and it returns the poke to the one that made the listing

**Comment by OfficialSqu1dz** [comment]
conclusion is basically just want something to use to remove GTS listings while giving it back to said person safely

**Comment by NickImpact** [comment]
Yes, admin functionality is planned, but GTS updates are on hold until my schedule frees up or a demanding update/fix is required for plugin functionality. Also, you can easily eliminate underselling with minimum prices, it is what these were designed for after all.

**Comment by OfficialSqu1dz** [comment]
Gotcha, thanks for the info though :)

**Comment by Eufranio** [comment]
+1 for this. Admins could cancel an listing by midde-clicking the listing item in the UI.

**Comment by NickImpact** [comment]
Coming in 6.0.0.
